File: NewsDemo/NewsCache+CoreDataProperties.m
//
//  NewsCache+CoreDataProperties.m
//  
//
//  Created by Seven on 2017/6/28.
//
//

#import "NewsCache+CoreDataProperties.h"

@implementation NewsCache (CoreDataProperties)

+ (NSFetchRequest<NewsCache *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"NewsCache"];
}

@dynamic ad_id;
@dynamic commentNum;
@dynamic elapseTime;
@dynamic feedsType;
@dynamic from;
@dynamic group_id;
@dynamic images;
@dynamic isBigPic;
@dynamic item_id;
@dynamic log_extra;
@dynamic runtime;
@dynamic tag;
@dynamic title;
@dynamic url;

@end

File: NewsDemo/NewsCache+CoreDataProperties.swift
//
//  NewsCache+CoreDataProperties.swift
//  
//
//  Created by Seven on 2017/6/28.
//
//

import Foundation
import CoreData


extension NewsCache {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<NewsCache> {
        return NSFetchRequest<NewsCache>(entityName: "NewsCache")
    }


}
